### PR TITLE
Fix missing identifier check in ace-permissions and jobsync modules

### PR DESCRIPTION
**Pull request by Jordan2139**

### CODE REVIEW

File: sonorancms/server/modules/ace-permissions/ace-permissions_sv.lua [hunk 1]
```diff
@@ -162,6 +167,11 @@ function initialize()
 		payload['key'] = Config.APIKey
 		payload['type'] = 'GET_ACCOUNT_RANKS'
 		payload['data'] = {{['apiId'] = identifier}}
+		if identifier == nil then
+			TriggerEvent('SonoranCMS::core:writeLog', 'warn', 'Player ' .. GetPlayerName(src) .. ' was denied access due to not having a ' .. Config.apiIdType .. ' identifier.')
+			TriggerClientEvent('chat:addMessage', src, {color = {255, 0, 0}, multiline = true, args = {'SonoranPermissions', 'You must have a ' .. Config.apiIdType .. ' identifier to use this command.'}})
```
Comment (Fireant456) [inline sonorancms/server/modules/ace-permissions/ace-permissions_sv.lua:172]: Change "SonoranPermissions" to "SonoranCMS"

File: sonorancms/server/modules/jobsync/jobsync_sv.lua [hunk 2]
```diff
@@ -241,6 +246,11 @@ function initialize()
 		payload['key'] = Config.apiKey
 		payload['type'] = 'GET_ACCOUNT_RANKS'
 		payload['data'] = {{['apiId'] = identifier}}
+		if identifier == nil then
+			TriggerEvent('SonoranCMS::core:writeLog', 'warn', 'Player ' .. GetPlayerName(src) .. ' was denied access due to not having a ' .. Config.apiIdType .. ' identifier.')
+			TriggerClientEvent('chat:addMessage', src, {color = {255, 0, 0}, multiline = true, args = {'SonoranPermissions', 'You must have a ' .. Config.apiIdType .. ' identifier to use this command.'}})
```
Comment (Fireant456) [inline sonorancms/server/modules/jobsync/jobsync_sv.lua:251]: Change "SonoranPermissions" to "SonoranCMS"

**Review by Fireant456 — CHANGES_REQUESTED** [review]
Please see in line comments for quick revisions.

**Review by Fireant456 — APPROVED** [review]
QA Pass, but waiting on https://app.clickup.com/t/867965dc5 (CMS Core Updater logic rework to not overwrite module config files) for release

**Review by DawsonGodin — APPROVED** [review]

**Review by DawsonGodin — APPROVED** [review]

**Pull request merged**